tests/integration_tests/test_experiment.py

```diff
@@ -327,12 +329,13 @@ def test_experiment_image_inputs(image_parms: ImageParms, csv_filename: str):
     input_features[0]['encoder'] = image_parms.image_encoder
     input_features[0]['preprocessing'][
         'in_memory'] = image_parms.in_memory_flag
-    rel_path = generate_data(input_features, output_features, csv_filename)
+    rel_path = generate_data(input_features, output_features, os.path.join(tmpdir,csv_filename))
```

Comment:
The project is following PEP8 style guidelines.  In particular there should be a space following a comma, e.g., `os.path.join( tmpdir, csv_filename)`